src/test/opentracing_api.ts

```diff
@@ -0,0 +1,79 @@
+
+import { expect } from 'chai';
+import * as opentracing from '../index';
+
+export function opentracingAPITests(): void {
```

Comment:
Tbh I don't see any value in these tests. They just verify that exports and methods exist. This is nothing the compiler can't warn you about.